packages/nc-gui/components/smartsheet/toolbar/SearchData.vue

```diff
@@ -90,7 +94,12 @@ watchDebounced(
       @click="isDropdownOpen = !isDropdownOpen"
     >
       <GeneralIcon icon="search" class="ml-1 h-3.5 w-3.5 text-gray-500 group-hover:text-black" />
-
+      <NcBadge color="purple">
+      <p class="pt-1 text-center text-xs font-regular text-gray-400"
```

Comment:
1. Use `truncate` which gives `overflow: hidden; text-overflow: ellipsis; white-space: nowrap;` to truncate overflowing text with an ellipsis (…) if needed.
2. I think `font-regular` makes no impact and is unnecessary here.